src/components/VersionModal/index.tsx

```diff
@@ -31,19 +31,15 @@ const VersionModal: React.FC<VersionModalProps> = ({ setModalVisible }) => {
             width={425}
         >
             <div>
-                {" "}
-                Application:{" "}
+                Application:
                 <span className={styles.blueText}>
-                    {" "}
-                    simularium-website v{SIMULARIUM_WEBSITE_VERSION}{" "}
+                    simularium-website v{SIMULARIUM_WEBSITE_VERSION}
                 </span>
             </div>
             <div>
-                {" "}
-                Viewer:{" "}
+                Viewer:
```

Comment:
you actually need a space here. now it looks like `Application:simularium-website v0.10.0`